tests/openadapt/test_layout.py

```diff
@@ -0,0 +1,97 @@
+from typing import List
+from openadapt.models import Recording
+from PIL import Image
+from transformers import pipeline
+import numpy as np
+
+
+class TestQAPipeline:
+    def __init__(self, image_file_paths: List[str]):
+        self.image_list = [
+            Image.open(img_file_path).convert("RGB")
+            for img_file_path in image_file_paths
+        ]
+
+    def document_query(self, image: np.ndarray, question: str) -> str:
```

Comment:
Why not test `LayoutExtractionReplayStrategyMixin.document_query` directly? It seems we are duplicating functionality here.

Reply:
```BaseReplayStrategy``` has an abstract method, get_next_action_event and so it threw an error whenever I tried to use the class because I didn't implement it in ```LayoutExtractionReplayStrategyMixin```..not sure why I didn't just implement and return a call to the base class's method in the first place but I've added the necessary changes to get this working 👍